csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -4058,15 +4503,15 @@ The entities ("conformance targets") for which this document defines requirement
 A text file satisfies the "CSAF document" conformance profile if the text file:
 
 * conforms to the syntax and semantics defined in section 3.
-* does not fail any mandatory test defined in section 4.1.
-* satisfies at least one profile defined in section 5.
+* satisfies at least one profile defined in section 4.
+* does not fail any mandatory test defined in section 5.1.
 
 ### 8.1.2 Conformance Clause 2: CSAF producer
 
 A program satisfies the "CSAF producer" conformance profile if the program:
 
-* produces output in the CSAF format, according to the syntax and semantics defined in section 3. The output MUST not fail any mandatory test defined in section 4.1.
-* satisfies those normative requirements in section 3 and 6 that are designated as applying to CSAF producers.
+* produces output in the CSAF format, according to the syntax and semantics defined in section 3. The output MUST satisfies at least one profile defined in section 4 and MUST NOT not fail any mandatory test defined in section 5.1.
```

Comment:
@sthagen I was wondering whether we could reference Conformance Clause 1 here. I guess that would be easier to read.

Reply:
Back referencing a conformance clause inside the conformance section itself is very good service to the reader and IMO fully compatible with any OASIS expectation on the conformance section. Please go ahead with that approach. Thanks for identifying. Does that mean in this case that conformance clause 2 extends clause 1?

Reply:
What about the following:

```suggestion
* produces output in the CSAF format, according to the conformance profile "CSAF document" .
```

However, I guess we should remove the text file part in conformance clause 1.